prow/scripts/cluster-integration/pr-gke-integration.sh

```diff
@@ -24,71 +44,107 @@ trap cleanup EXIT
 
 #!Put cleanup code in this function!
 cleanup() {
+    #Try to preserve exit status unless a new error occurs
+    EXIT_STATUS=$?
+
     #Turn off exit-on-error so that next step is executed even if previous one fails.
     set +e
 
     if [ -n "${CLEANUP_CLUSTER}" ]; then
```

Comment:
Please make it more obvious if an error occurred and the cleanup is started. It is not easy to see from the logs that there was an error. An example failure: http://35.234.112.122/log?job=prow/kyma/kyma-integration&id=1064501075270176768

Reply:
Added explicit notification about error.